### PR TITLE
docs: Translation-triggered changes.

### PR DESCRIPTION
**Pull request by smoe**
Some tricky ones on Gscreen - sidenote, a capital G (not enforced in patch) seems to be dominant in the spelling of Gscreen (and I like it).

### CODE REVIEW

File: docs/src/gui/gscreen.adoc [hunk 1]
```diff
@@ -499,19 +499,25 @@ skin folder.
...
-yes/no messages require the user to select yes or no buttons to close the dialog. +
-yes/no messages have three HAL pins - one to show the dialog, one for waiting, +
-and one for the answer. +
+These are defined in the INI file and controlled by HAL pins:
+
+Boldtext:: is generally a title.
+text:: is below that and usually longer.
+Detail:: is hidden unless clicked on.
+pinname:: is the basename of the HAL pins.
+type:: specifies whether its a yes/no, ok, or status message
```
Comment (hansu) [inline docs/src/gui/gscreen.adoc:508]:
```suggestion
MESSAGE_BOLDTEXT:: is generally a title.
MESSAGE_TEXT:: is below that and usually longer.
MESSAGE_DETAILS:: is hidden unless clicked on.
MESSAGE_PINNAME:: is the basename of the HAL pins.
MESSAGE_TYPE:: specifies whether its a yes/no, ok, or status message
```

To make it really clear

**Comment by hansu** [comment]
It would be nice if you would not always add new commits to once published PR's. It's not very nice to the reviewer as he always has to start again from the beginning.

**Comment by hansu** [comment]
Okay I understand the problem. 

>What I do not do is that I squash or --amend, 

But I just saw: @[smoe](https://github.com/smoe) smoe [force-pushed](https://github.com/LinuxCNC/linuxcnc/compare/61131fa82e2418475e933d1764122d1a5bb9325f..4766475bf72149ea07f86510df014dec02707c97) the VSCode_on_Mac_with_smallish_changes branch from[ 61131fa ](https://github.com/LinuxCNC/linuxcnc/commit/61131fa82e2418475e933d1764122d1a5bb9325f)to[ 4766475 ](https://github.com/LinuxCNC/linuxcnc/commit/4766475bf72149ea07f86510df014dec02707c97)2 hours ago

However, in the worst case the PR gets never merged, because:
You make changes, then I review and I wait for the build to finish. But before the build finishes you add more commits, I review and wait again for the build and so on.

I would suggest not to push further changes to the remote (or this branch) once you have published the PR.
You can still add commits on another branch, no problem. This branch could start at the head of current branch. So if the PR is accepted without changes, there will be no conflicts.

**Comment by smoe** [comment]
> Okay I understand the problem.
> 
> > What I do not do is that I squash or --amend,
> 
> But I just saw: @[smoe](https://github.com/smoe) smoe [force-pushed](https://github.com/LinuxCNC/linuxcnc/compare/61131fa82e2418475e933d1764122d1a5bb9325f..4766475bf72149ea07f86510df014dec02707c97) the VSCode_on_Mac_with_smallish_changes branch from[ 61131fa ](https://github.com/LinuxCNC/linuxcnc/commit/61131fa82e2418475e933d1764122d1a5bb9325f)to[ 4766475 ](https://github.com/LinuxCNC/linuxcnc/commit/4766475bf72149ea07f86510df014dec02707c97)2 hours ago

Yes. When I saw your addition (which could have been a separate PR) I considered this ready from your side and rebased on master. Because of that rebase I had to push-force. And I did that rebase since I had to integrate your changes to my branch with my further local changes in a serial manner.

> However, in the worst case the PR gets never merged, because: You make changes, then I review and I wait for the build to finish. But before the build finishes you add more commits, I review and wait again for the build and so on.

Well, at some point we are at some 100% translation coverage or I change the machine again. I do not have trouble with delays except that I encounter outdated po4a entries in weblate.

> I would suggest not to push further changes to the remote (or this branch) once you have published the PR. You can still add commits on another branch, no problem. This branch could start at the head of current branch. So if the PR is accepted without changes, there will be no conflicts.

My immediate thought is that you do not review until it builds. If the build is fine then just accept it. The last set of changes from your side where not because of my changes but because of what was in the documentation before, which then triggered a rebuild when I accepted those.

And, as you may guess, just like you continued editing on that branch, so did I locally.

I sense this comes down to "we need shorter build times". Or just live with how it is.

**Pull request merged**